include/revng/Pipes/ToolCLOptions.h

```diff
@@ -48,7 +48,12 @@ class ToolCLOptions {
                             PipelineManager &Manager) {
     const auto &Name = ModelGlobalName;
     auto *Model(cantFail(Manager.context().getGlobal<ModelGlobal>(Name)));
-    return Model->loadFromDisk(ModelOverride);
+
+    revng_assert(!isURLS3(ModelOverride));
```

Comment:
1. Why? I see many places using this function that should be storage-backend agnostic.
2. Rename to `isS3URL`.

Reply:
1. It was to prevent "surprising" accidental usage by command-line tools, but with the introduction of `Path::fromLocalStorage` this is not needed anymore
2. Done